packages/@angular/cli/models/webpack-configs/common.ts

```diff
@@ -67,7 +67,7 @@ export function getCommonConfig(wco: WebpackConfigOptions) {
     devtool: buildOptions.sourcemaps ? 'source-map' : false,
     resolve: {
       extensions: ['.ts', '.js'],
-      modules: [nodeModules],
+      modules: ['node_modules', nodeModules],
```

Comment:
It provides a fallback to the project's node modules.  Useful for linked modules with peer dependencies.  When such a package is installed normally the peer dependencies would normally be located in the project's node modules.